package-coverage/generator/coverage.go

```diff
@@ -129,8 +129,11 @@ var execCoverage = func(dir string, coverageFilename string) error {
 		"-coverprofile=" + coverageFilename,
 	}
 
+	arguments = append(arguments, goTestArgs...)
+
 	cmd := exec.Command(command, arguments...)
 	cmd.Dir = dir
+	cmd.Stdout = os.Stdout
```

Comment:
Yes, I was thinking of that. That should allow the "quietness" to be preserved, unless we also want `go test -v`; I'll fix.

Or, would you prefer a -vv for seeing the output of `go test`, where `-vv` implies `-v`?